### PR TITLE
Fix styled component errors

### PR DESCRIPTION
**Pull request by lieut-data**
## Summary
Several styled components were being created on each render: avoid this (and squelch the corresponding console log).

## Ticket Link
N/A

### CODE REVIEW

File: webapp/src/components/backstage/playbook_runs/playbook_run/rhs_info_overview.tsx [hunk 1]
```diff
@@ -266,21 +266,25 @@ interface ItemProps {
     onClick?: () => void;
 }
 
+const StyledIcon = styled.div`
```
Comment (lieut-data) [inline webapp/src/components/backstage/playbook_runs/playbook_run/rhs_info_overview.tsx:269]: I think this creates an extra DOM element -- any suggested improvements?

Comment (trilopin) [reply webapp/src/components/backstage/playbook_runs/playbook_run/rhs_info_overview.tsx:269]: you could remove this one and set at OverviewitemName if you are concerned about it

```ts
const OverviewItemName = styled.div`
    display: flex;
    align-items: center;
    svg {
        margin-right: 11px;
    }
`;
```

**Review by trilopin — APPROVED** [review]
nice catch @lieut-data

File: webapp/src/components/backstage/playbook_runs/playbook_run/rhs_info_overview.tsx (hunk 1 above)
Comment (calebroseland) [reply webapp/src/components/backstage/playbook_runs/playbook_run/rhs_info_overview.tsx]: The container is already `flex`, so I think `gap` would be the most straightforward:
```diff
 const OverviewItemName = styled.div`
     display: flex;
     align-items: center;
+    gap: 11px;
 `;
```

**Review by calebroseland — APPROVED** [review]

**Pull request merged**